### PR TITLE
Update template according to new specification

### PR DESCRIPTION
**Pull request by famosab**
## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

I updated the modules template according to [this PR](https://github.com/nf-core/modules/pull/5281). This way of handing the test data was enforced during the march 2024 hackathon and I thought it should be reflected in the template 😄

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
Can you add a CHANGELOG entry in the TEMPLATE section

**Comment by famosab** [comment]
How excatly do I need to add this? as a new patch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # nf-core/tools: Changelog
 
+
```
Comment (maxulysse) [inline CHANGELOG.md:3]:
```suggestion
```

**Comment by famosab** [comment]
There is still one check failing ... :(

**Comment by maxulysse** [comment]
@mirpedrol any idea?

**Review by mirpedrol — APPROVED** [review]
The failing test was due to a Docker error, it is fixed now, I re-run the tests so it should be ok once they finish running

**Comment by mirpedrol** [comment]
All green now ✅ thanks!

**Pull request merged**